starneto_odometry_converter_node:
  ros__parameters:
    mgrs_precision: 4
    allow_status_map: [false, false, false,
                       false, true, true,
                       false, false, false,
                       false
                      ]